cli/cmd/initialize/wizard/survey_answers.go

```diff
@@ -14,20 +14,22 @@ type surveyAnswers struct {
 	RequirementsFile string
 }
 
-func (s surveyAnswers) appendAnswersToApp(a *tool.Tool) {
-	a.Name = s.Name
-	a.Description = s.Description
-	a.Library, _ = tool.GetLibraryByName(s.LibraryName)
-	a.AppFile = strings.Trim(s.AppFile, " 	")
-	a.RequirementsFile = strings.Trim(s.RequirementsFile, " 	")
+func (s surveyAnswers) updateManifest(m *manifest.Manifest) {
+	lib, _ := manifest.GetLibraryByName(s.LibraryName)
+	m.Name = s.Name
+	m.Description = s.Description
+	m.Library = lib
+	m.AppFile = strings.Trim(s.AppFile, " 	")
+	m.RequirementsFile = strings.Trim(s.RequirementsFile, " 	")
```

Comment:
It only trims at the ends. If you want to use a file with a name that starts with a space or a tab, then it's really not our fault :smile:

If the user adds whitespace by accident it might be nice that it is trimmed.